image_services/rtc_services/hkh/hkh_wm.json

```diff
@@ -0,0 +1,23 @@
+{
+  "project_name": "HKH",
+  "s3_prefix": "HKH/",
+  "dataset_name": "WM",
+  "raster_filter": "*_WM.tif",
+  "raster_function_templates": [
+    "WatermapExtentBlueOnly.rft.xml",
+    "WatermapExtentWhiteandBlue.rft.xml"
+  ],
+  "default_raster_function_template": "WatermapExtentBlueOnly.rft.xml",
+  "service_folder": "HKH",
+  "service_name": "HKH_WM",
```

Comment:
The existing service name is `HKH_Watermap_Extent` : https://asf.img.arcgis.com/arcgis/rest/services/HKH/HKH_Watermap_Extent/ImageServer